src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/RamDeliusAPIConvictionService.kt

```diff
@@ -0,0 +1,73 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken
+import org.springframework.stereotype.Service
+import org.springframework.web.util.UriComponentsBuilder
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.component.RestClient
+import java.time.LocalDate
+
+data class Sentence(
+  val description: String,
+  val expectedEndDate: LocalDate,
+)
+
+data class Offence(
+  val category: String,
+  val subCategory: String,
+)
+
+data class Conviction(
+  val id: Long,
+  val date: LocalDate,
+  val sentence: Sentence,
+  val mainOffence: Offence,
+  val active: Boolean,
+)
+
+data class ConvictionDetails(
+  val conviction: Conviction,
+)
+
+@Service
+class RamDeliusAPIConvictionService(
```

Comment:
We already have a `uk.gov.justice.digital.hmpps.hmppsinterventionsservice.component.RamDeliusClient` class which I believe we can use that.

Reply:
Pushed changes to use RamDeliusClient as requested @kth13